guides/release/getting-started/quick-start.md

```diff
@@ -366,4 +383,29 @@ add the following directive to the application's virtual host configuration:
 FallbackResource index.html
 ```
 
-<!-- eof - needed for pages that end in a code block  -->
+## Next steps
+
+Now that your app is deployed, what should you do next?
+
+### Advance to the next level
+
+There is an official, free tutorial here in the Guides that delves deeper into some of the features you used today.
+[Give it a try!](../tutorial/part-1)
```

Comment:
```suggestion
[Give it a try!](https://guides.emberjs.com/release/tutorial)
```
this is still not going to the correct place, so rather than keep fiddling with it, I'm putting the whole link in for now.